### PR TITLE
Update fixed-hash & uint utilities

### PR DESCRIPTION
**Pull request by Robbepop**
Updates the utilities
- `fixed-hash` from version `0.2` to `0.3` (beta)
- `uint` from version `0.4` to `0.5` (beta)

Refer to the following links for further information about their updates
- `fixed-hash`
    - `0.3.0-beta.0`: https://github.com/paritytech/parity-common/pull/71
    - `0.3.0-beta.x`: https://github.com/paritytech/parity-common/pull/76
- `uint`
    - `0.5.0-beta.x`: https://github.com/paritytech/parity-common/pull/77

### CODE REVIEW

File: core/state-db/src/lib.rs [hunk 1]
```diff
@@ -32,16 +32,16 @@
 //!
 // end::description[]
 
-#[macro_use] extern crate log;
-#[macro_use] extern crate parity_codec_derive;
+#[macro_use]extern crate log;
```
Comment (bkchr) [inline core/state-db/src/lib.rs]: Nitpick: Either move the `macro_use` above or re-add the whitespace.

Comment (Robbepop) [reply core/state-db/src/lib.rs]: done

**Comment by Robbepop** [comment]
Fixed failing CI tests.

**Comment by pepyakin** [comment]
And btw, welcome to the substrate repo @Robbepop !

**Pull request merged**